python/labours/pb_pb2.py

```diff
@@ -2,11 +2,13 @@
 # source: pb.proto
 
 import sys
-_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+
```

Comment:
We can discard this change

Reply:
Ah yeah, probably an automated change made by PyCharm's refactoring bits. Sorry for the noise.